slack_bot/slash_commands/answears.py

```diff
@@ -0,0 +1,41 @@
+import json
+from enum import Enum
+from slack_bot.settings import env_settings
+
+
+class Commands(Enum):
+    gen_job_desc = "generate_job_description"
+    create_media_post = "create_social_media_post"
+    match_resumes = "match_resumes"
+    scan_resume = "scan_resume"
+    get_chats = "get_chats"
+
+
+def get_answear(command: str):
+    with open(env_settings.bot_data_path, 'r') as f:
+        data = json.load(f)
```

Comment:
error handling if json is not correct or doesn't exist?

Reply:
will be done later, along with functionality to modify it

Reply:
please create separate issue for "do-later" taff and leave the comment with issue number here so we will have it linked to the issue.

Reply:
https://github.com/COXIT-CO/CVScanAI/issues/7

Reply:
#7 is issue to make bot promts easy configurable, do you plan to do it in scope of that issue? I meant to create some separate cleanup kind of task and leave all the comments what should be refactored under that issue.

Reply:
I plan to do it in the scope of that issue. I described everything in the description.